core/templates/dev/head/pages/exploration_editor/ParamChangesEditorDirective.js

```diff
@@ -83,7 +83,8 @@ oppia.directive('paramChangesEditor', [function() {
               angular.copy(DEFAULT_PARAM_SPEC));
             $scope.paramNameChoices = generateParamNameChoices();
           }
-          $scope.paramChangesService.displayed.push(newParamChange);
+          $scope.paramChangesService.displayed.push(
+            angular.copy(newParamChange));
```

Comment:
I actually don't understand why I need to copy here, but if I don't then the generatorIds of the paramChanges are coupled. If anyone cares to enlighten me that'd be great